### ISSUE TITLE
G68?

### ISSUE THREAD

**Issue by WackoKacko** [vladbabii/klippy-cnc#6]
I would love to be able to rotate the relative coordinate system about the Z-axis.

**Comment by DeltaMaker** [comment]
@WackoKacko, I started implementing the G68 and G69 g-codes a couple months ago, but haven't had time to do much testing. As I recall, my initial tests where successful. Comments are welcome.
https://github.com/DeltaMaker/klipper/tree/work-g68-20220617

**Comment by WackoKacko** [comment]
@DeltaMaker I copied over your modified g68 klippy/extras folder to my printer, restarted firmware, and it didn't break. G68 recognizes when I give it values outside the range +/-360, but it also gives me a M400 error when I try to use it correctly. I've been trying with the syntax "G68 R180" and "G68 X0 Y0 R180", to be clear, but I don't think this is the issue. Calling M400 before and/or after G68 makes no difference.
Any tips?

**Comment by DeltaMaker** [comment]
@WackoKacko Yes, "G68 R180" or "G68 X0 Y0 R180" is the expected syntax. Can you provide the more details regarding the gcodes you are sending immediately before and after the G68, and the exact error you are receiving?

**Comment by WackoKacko** [comment]
Hi, @DeltaMaker. It turns out G68 was working exactly as it was supposed to. The problem I was having was with polar kinematics. Klipper can't bear it when I give it command G1 X10 Y0 and then G1 X-10 Y0. It gives me an error, stops print, I think it disconnects mcu or smthn like that, too. G68 works great. Now, I just need G10, and G53-59 :)
